reference/shaderpart.md

```diff
@@ -0,0 +1,72 @@
+## ShaderPart
```

Comment:
Personally I prefer the name ShaderProgram even if it is different from X3DOM name.
But this is also fine.